packages/minifiers/minifiers.js

```diff
@@ -133,3 +133,24 @@ CssTools = {
   }
 };
 
+var toOSPath = function (p) {
```

Comment:
But don't we need these functions to uniload something? Maybe this is another package version parser situation